EmaPajic/assignment4/test_assignment4.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+import unittest
+from assignment4 import count_islands
+
+class Test(unittest.TestCase):
+    
+    def setUp(self):
+        self.tiles_empty = []
+        self.tiles1 = [[False,False],[False,False]]
+        self.tiles2 = [[True,True]]
+        self.tiles3 = [[True,True,True,True,True],[True,False,False,False,True],[True,False,True,False,True],[True,False,False,False,True],[True,True,True,True,True]]
```

Comment:
Please put the different sub-arrays each on a different line and use spacing, so it's easier to understand where are the islands in your example.
E.g.
```
[[True , True , True , True , True],
 [True , False, False, False, True],
 [True , False, True , False, True],
 [True , False, False, False, True],
 [True , True , True , True , True]]
```

Reply:
updated :)